<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vsqtopencv01Class</class>
 <widget class="QWidget" name="vsqtopencv01Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vsqtopencv01</string>
  </property>
  <widget class="QPushButton" name="open_image">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>30</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_video">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>150</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>打开视频</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_camera">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>280</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="identify">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>430</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>识别</string>
   </property>
  </widget>
  <widget class="QLabel" name="video_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1051</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="takepic">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>570</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>拍照</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>860</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vsqtopencv01.qrc"/>
 </resources>
 <connections/>
</ui>
